diff --git a/src/debug.h b/src/debug.h
index 2cfe8e1..2f96d3a 100644
--- a/src/debug.h
+++ b/src/debug.h
@@ -27,10 +27,6 @@
 # define ASSERT(left, operator, right) (void(0))
 #else
 
-#if !defined(_WIN32) && !defined(__APPLE__) && !defined(__ANDROID__)
-#include <execinfo.h>
-#endif
-
 template<typename T, typename U>
 void _on_assert_fail(const char* vara, const char* op, const char* varb,
                      T a, U b, const char* file, int line)  {
@@ -38,16 +34,6 @@
       vara << "[" << a << "] " << op << " " << varb << "[" << b << "]" <<
       std::endl;
 
-#if !defined(_WIN32) && !defined(__APPLE__) && !defined(__ANDROID__)
-  void *callstack[64];
-  size_t size;
-  size = backtrace(callstack, 64);
-  char** strings = backtrace_symbols(callstack, size);
-  for (size_t i=0; i<size; i++) {
-    std::cerr << strings[i] << std::endl;
-  }
-  free(strings);
-#endif
   exit(1);
 }
